### ISSUE TITLE
Changing username of people connected over EU login?

### ISSUE THREAD

**Issue by D063520** [ec-doris/EuKnowledgeGraph#131]
The users that connect over eu-login are quite cryptic:

<img width="1405" alt="Screenshot 2023-03-27 at 16 21 07" src="https://user-images.githubusercontent.com/10883585/227967925-ef905242-9c9e-473a-9004-fe992a4b0579.png">

What about grabbing as a username something different from EU Login. Here some possible infomation like "Name" or "e-mail"

<img width="1405" alt="Screenshot 2023-03-27 at 16 19 46" src="https://user-images.githubusercontent.com/10883585/227967767-4826a952-eea0-4655-9893-d2768d170ca0.png">

**Comment by madewild** [comment]
@raphdom do we get the name or not?

**Comment by raphdom** [comment]
no, we should change the way that we are doing the login, using the open id, with that we can get all the info easily...
